Build/scripts/pogo_training.ps1

```diff
@@ -0,0 +1,45 @@
+param (
+    [string[]]$scenarios = @(),
+
+    [Parameter(Mandatory=$True)]
+    [string]$binary,
+
+    [Parameter(Mandatory=$True)]
+    [string]$arch,
+
+    # force callers to specify this in case of future use
+    [Parameter(Mandatory=$True)]
+    [string]$flavor = ""
+)
+
+$binpath = Split-Path -Path $binary -Parent
+$pgodll = Join-Path $binpath "pgort140.dll";
+if (-not (Test-Path ($pgodll))) {
+    $vcarchpath = ""
+    if ($arch -eq "x64") {
+        $vcarchpath = "amd64"
+    } elseif ($arch -eq "arm") {
+        $vcarchpath = "arm"
+    }
+    $pgoSrcDll = Join-Path ${env:ProgramFiles(x86)} (Join-Path (Join-Path "Microsoft Visual Studio 14.0\VC\bin" $vcarchpath) "pgort140.dll")
```

Comment:
For this one I have a better fix using property sheet. It will be done at a later point